src/sunstone/etc/sunstone-views/vcenter/user.yaml

```diff
@@ -6,925 +6,979 @@ confirm_vms: false
+      Network.reserve_dialog: true
+      Network.addtocluster: false
+      Network.rename: true
+      Network.chown: false
+      Network.chgrp: false
+      Network.chmod: true
+      Network.delete: true
+      Network.hold_lease: true
+      Network.release_lease: true
+      Network.add_ar: false
+      Network.remove_ar: true
+      Network.update_ar: true
+      Network.lockU: true
+      Network.unlock: true
+      Network.edit_labels: true
+      Network.menu_labels: true
+      Network.add_secgroup: true
+      Network.remove_secgroup: true
+      Network.recover: false
+  vnets-templates-tab:
+    panel_tabs:
+      vnet_template_info_tab: true
+      vnet_template_ar_list_tab: true
+      vnet_template_sg_list_tab: true
+      vnet_template_clusters_tab: true
+      vnet_template_template_tab: true
+    table_columns:
+      - 0 # Checkbox
+      - 1 # ID
+      - 2 # Name
+      - 3 # Owner
+      - 4 # Group
+      - 6 # Cluster
+      #- 10       # Labels
+      #- 11       # Search data
+    actions:
+      VNTemplate.refresh: true
+      VNTemplate.create_dialog: false
+      VNTemplate.instantiate_vnets: true
+      VNTemplate.update_dialog: true
+      VNTemplate.rename: true
+      VNTemplate.chown: true
+      VNTemplate.chgrp: true
+      VNTemplate.chmod: true
+      VNTemplate.delete: true
+      VNTemplate.hold_lease: true
+      VNTemplate.release_lease: true
+      VNTemplate.add_ar: true
+      VNTemplate.remove_ar: true
+      VNTemplate.update_ar: true
+      VNTemplate.update: true
+      VNTemplate.lockU: true
+      VNTemplate.unlock: true
+      VNTemplate.edit_labels: true
+      VNTemplate.menu_labels: true
+  vnets-topology-tab:
+    panel_tabs:
+    actions:
+      NetworkTopology.refresh: true
+      NetworkTopology.fit: true
+      NetworkTopology.collapseVMs: true
+      NetworkTopology.openVMs: true
+  secgroups-tab:
+    panel_tabs:
+      security_group_info_tab: true
+      security_group_vms_tab: true
+    table_columns:
+      - 0 # Checkbox
+      - 1 # ID
+      - 2 # Owner
+      - 3 # Group
+      - 4 # Name
+      #- 5         # Labels
+      #- 6        # Search data
+    actions:
+      SecurityGroup.refresh: true
+      SecurityGroup.create_dialog: true
+      SecurityGroup.update_dialog: true
+      SecurityGroup.rename: true
+      SecurityGroup.chown: true
+      SecurityGroup.chgrp: true
+      SecurityGroup.chmod: true
+      SecurityGroup.clone_dialog: true
+      SecurityGroup.commit_dialog: true
+      SecurityGroup.delete: true
+      SecurityGroup.edit_labels: true
+      SecurityGroup.menu_labels: true
+  settings-tab:
+    panel_tabs:
+      settings_info_tab: true
+      settings_config_tab: false
+      settings_quotas_tab: true
+      settings_group_quotas_tab: true
+      settings_accounting_tab: true
+      settings_showback_tab: true
+      settings_auth_tab: true
+    actions:
+      # Buttons for settings_info_tab
+      User.update_password: true
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets can allow attackers to bypass authentication methods and perform malicious actions. Possible remediation approaches are:

	 1. Manage secret data using environment variables:
 		 a. Identify the secret data and store that data in environment variables on your local and production server. The setting process may vary depending on your OS and environment.
		 b. Replace hard-coded secrets with references to environment variables, e.g., `password = os.environ.get(&#39;PASSWORD&#39;)`.
	 2. Use AWS Secrets Manager to store, rotate, monitor, and control access to secrets. To retrieve secrets, you can replace hardcoded secrets in applications with a call to Secrets Manager APIs. To use Secrets Manager:
		 a. Visit [Secrets Manager](https://aws.amazon.com/secrets-manager/) on the AWS Management Console.
		 b. Choose the secret type on the console page and follow the instructions.
		 c. Use the code samples suggested in the console to retrieve the secret in your application.